monitoring/influxdb.go

```diff
@@ -105,52 +210,141 @@ type influxdbMonitor struct {
 	client   influxdb.Client
 	database string
 	logger   log.Logger
+
+	pointChan          chan *influxdb.Point
+	batchWriteInterval time.Duration
+	cacheEvent         *cacheEvent
 }
 
-// InsertRecord part of monitor.Monitor.
-func (im influxdbMonitor) InsertRecord(measurement string, value interface{}, tags map[string]string, fields map[string]interface{}, at time.Time) {
-	if fields == nil {
-		fields = map[string]interface{}{}
+type cacheEvent struct {
```

Comment:
I'm having trouble understanding the purpose of this struct. Can you please add some comments to describe how it is used?

Reply:
Removed `cacheEvent` in https://github.com/theplant/appkit/pull/25/commits/af37ccc7faaceb0087dc23b229d3911c1ad74011.